code/modules/events/immovable_rod.dm

```diff
@@ -11,7 +11,7 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	announceWhen = 5
 
 /datum/event/immovable_rod/announce()
-	event_announcement.Announce("What the fuck was that?!", "General Alert")
+	event_announcement.Announce("Unidentified object inbound. Brace for impact.", "Collision Alert")
```

Comment:
You've never heard Comms Officer Jenkins, have you?
I also like the classic. Can't you have it do a pick() or something for one or the other?